client/src/components/Admin/OrganizationEdit/ContactDetails.js

```diff
@@ -65,36 +66,56 @@ export default function ContactDetails({
         <Grid item sm={6} xs={12}>
           <div>
             <Label
-              id="twitter"
+              id="twitter-label"
               label="Twitter"
-              tooltipTitle="URL must start with 'https://twitter.com/ or https://x.com/'"
-              href={values.twitter}
+              tooltipTitle="Enter your Twitter username"
+              href="https://twitter.com/"
+              handle={values.twitter}
             />
 
             <TextField
               id="twitter"
               name="twitter"
-              placeholder="Twitter"
+              placeholder="Twitter username"
               value={values.twitter}
               onChange={handleChange}
               onBlur={handleBlur}
               helperText={touched.twitter ? errors.twitter : ""}
               error={touched.twitter && Boolean(errors.twitter)}
+              InputProps={{
+                startAdornment: (
+                  <InputAdornment position="start">
+                    https://twitter.com/
+                  </InputAdornment>
+                ),
+              }}
```

Comment:
instead of `startAdornment`, I was thinking of something like this. We might need to create a custom component for it 😅 
<img width="496" alt="Screen Shot 2024-08-12 at 9 53 55 PM" src="https://github.com/user-attachments/assets/098df852-ce66-47ee-a0e5-49db438822df">

Reply:
![Screenshot 2024-08-13 at 10 00 12 PM](https://github.com/user-attachments/assets/d2306869-141b-4689-a1a3-a226c6880e1d)
I did the twitter similar to your inspiration image and based on the conversation we had and kept the pinterest field the original suggestion from the stand up. Which one do you prefer?

Reply:
I can also change the pinterest field to have less empty space between the url and the handle.

Reply:
nice! I think both could work and yeah maybe less space between the url and the handle 👌